stock-usecase/src/main/java/org/dhis2/android/rtsm/data/models/TransactionItem.kt

```diff
@@ -5,4 +5,5 @@ import org.dhis2.android.rtsm.data.TransactionType
 data class TransactionItem(
     val icon: Int,
     val transactionType: TransactionType,
+    val transactionTypelabel: String,
```

Comment:
I think `label` should be fine as the class is already called TranscationItem, maybe `transactionType` will be enough with `type` as well...